react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+class Stopwatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      player: true,
+      timer: 0,
+      myVar: ''
+    };
+    this.handlePlayer = this.handlePlayer.bind(this);
+    this.handleStop = this.handleStop.bind(this);
+    this.handleClear = this.handleClear.bind(this);
+  }
+
+  render() {
+    const Player = this.state.player;
```

Comment:
Variables / consts should not start with a capital letter. Names with capital letters should only be used for classes.